redbot/core/commands/commands.py

```diff
@@ -461,10 +461,22 @@ async def can_run(
             if not change_permission_state:
                 ctx.permission_state = original_state
 
+    def is_enabled(self, ctx) -> bool:
+        if not self.enabled:
+            return False
+        if ctx.guild:
+            if self._disabled_in.has(ctx.guild.id):
+                return False
+            if self._disabled_in.has(ctx.channel.id):
+                return False
```

Comment:
You check for channel here but both the enable_in and disable_in only take in a guild obj